datasets/us-hiring-rates-pandemic.yaml

```diff
@@ -0,0 +1,33 @@
+Name: Covid Job Impacts - US Hiring Data Since March 1 2020
+Description: Daily updates on the volume of US job listings filtered by geography industry job family and role; normalized to pre-covid levels
+Documentation: https://greenwichhr-covidjobimpacts.s3.us-east-2.amazonaws.com/ghr_data_specs_covid_public.pdf
+Contact: info@Greenwich.HR
+ManagedBy: "[Greenwich.HR](http://www.greenwich.hr/)"
+UpdateFrequency: Daily 
+Tags:
+  - economics
+  - financial markets
+  - market data
+License: |
+  This data is made available under a [Creative Commons license](https://creativecommons.org/licenses/by-nc-nd/4.0/).
+  This data is free to use for non-commercial purposes under the terms of the license. You must
+  ensure that you acknowledge the source of this data as prescribed by this license. Data or any
+  derivatives made using the data are for internal use only and may not be distributed or otherwise
+  shared.
+Resources:
+  - Description: Greenwich.HR daily US hiring data (normalized to March 1 2020)
+    ARN: arn:aws:s3:::greenwichhr-covidjobimpacts
```

Comment:
I am unable to list anything within this bucket. To be included on this site, the data should be publicly available (and listable). https://gist.github.com/jedsundwall/a5d07ae3971fb172cf923d8de4625536 shows an example of an S3 bucket policy that allows this.